app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
+
+    URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
The URL is correctly defined as a constant in uppercase, which is a good practice for readability and maintainability.